bin/liquidator/src/main.rs

```diff
@@ -234,15 +245,15 @@ async fn main() -> anyhow::Result<()> {
         forbidden_tokens: liq_config.forbidden_tokens.clone(),
     };
 
-    let mut rebalance_interval = tokio::time::interval(Duration::from_secs(30));
-    let (rebalance_trigger_sender, rebalance_trigger_receiver) = async_channel::bounded::<()>(1);
+    let (rebalance_trigger_sender, rebalance_trigger_receiver) = tokio::sync::mpsc::channel(1);
```

Comment:
what's the reasoning for switching to tokio::sync::mpsc? in the past I had issues where tokio's channels where very bare-bones and didn't have functionality I wanted. Possibly that has changed since.

Reply:
Didn't realized `async_channel` is multi producer "multi" consumer (single consume). 
Going back to that for rebalancing, and I'll probably update the PR to use it for TxTrigger also